cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -117,7 +117,7 @@ macro (build_addon target prefix libs)
                 if("${matchres}" EQUAL 0)
                   string(REPLACE " " ";" loop_var "${loop_var}")
                   list(GET loop_var 1 include_name)
-                  string(REGEX REPLACE "[<>\"]" "" include_name "${include_name}")
+                  string(REGEX REPLACE "[<>\"]|kodi/" "" include_name "${include_name}")
```

Comment:
rebranding is not limited to the UI. Also things like useragent, installation folder, app name etc need to change in code as well. So if a string is related f.e. to the installation folder, ideally some cmake wide variable should be used for that. If the string is however related to extension points that likely won't change on rebrands, hardcoding should be fine.
I was only bringing this up because we had a related discussion at DevCon if you guys remember.